paf-mvp-demo-express/src/index.ts

```diff
@@ -28,6 +28,12 @@ const relative = (dir: string) => join(__dirname, dir);
               res.setHeader('Access-Control-Allow-Origin', '*');
               res.setHeader('Cache-Control', 'public, max-age=604800');
             }
+            if (/(js)$/.test(filePath)) {
```

Comment:
FAR: You already have it in mind, but to put in writing, the regex should test for the file extension, i.e. including the dot `/\.js$/`